actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -92,8 +95,15 @@ class ActionCable.Connection
       @disconnected = false
       @subscriptions.reload()
 
-    close: ->
+    close: (event) ->
       ActionCable.log("WebSocket onclose event")
+      if event.code == 1006
+        ActionCable.log("WebSocket received abnormal close error (" + event.code + "): " + event.reason)
```

Comment:
Also, why an additional log for `1006` only? Might as well log `event.code`, `event.reason`, and `event.wasClean` with every `close`.